README.md

```diff
@@ -20,3 +20,89 @@ For information on how to create a custom function, please refer to the [documen
 
 By providing a centralized repository for custom functions, the "catalog" repository enhances the flexibility and power of the Serverless Workflow DSL, enabling users to create more complex and tailored workflow solutions.
 
+## Structure
+
+Cataloged functions must follow the following structure:
+
+```
+./functions
+    /log                        👈 The name of the custom function. Must be lowercased and contain exclusively alphanumeric characters, with the exception of '-'.
+        /1.0.0                  👈 The semantic version of the custom function.
+            /function.yaml      👈 The function's definition.
+            /OWNERS.md          👈 Describes the owners/authors of the custom function.
+            /README.md          👈 Describes the custom function.
+        /2.0.0
+            ...
+```
+
+## Usage
+
+Below is an example of how to usea custom function in a Serverless Workflow:
+
+```yaml
+document:
+  dsl: '1.0.0'
+  namespace: samples
+  name: use-cataloged-function
+  version: '1.0.0'
+do:
+  - callCatalogedFunction:
+      call: https://github.com/serverlessworkflow/catalog/functions/log/1.0.0 #link to the repository directory that contains the function.yaml of the custom function to call.
+      #call: log:1.0.0 <--- this would also be a correct way to reference the function, as it is registered in the official catalog.
+      with:
+        message: Hello, world!
+```
+
+## Contributing
```

Comment:
It seems a duplication of `CONTRIBUTION.MD`? Why not have only the file and link it here?

Reply:
Good point!